model/task_queue_service_dependency.go

```diff
@@ -185,10 +237,11 @@ func (t *basicCachedDAGDispatcherImpl) rebuild(items []TaskQueueItem) error {
 		sort.SliceStable(su.tasks, func(i, j int) bool { return su.tasks[i].GroupIndex < su.tasks[j].GroupIndex })
 	}
 
+	dependsOnCache := make(map[string]bool)
```

Comment:
I tend to use map[string]struct{} rather than bools for sets.

Reply:
Ah, yes.